homework/index.js

```diff
@@ -27,21 +27,159 @@
         elem.setAttribute(key, value);
       }
     });
+
     return elem;
   }
 
+  function sortList() {
+    let i;
+    let switching;
+    let b;
+    let shouldSwitch;
+    const list = document.getElementById('select');
+    switching = true;
+    while (switching) {
+      switching = false;
+      b = list.getElementsByTagName('option');
+      for (i = 0; i < b.length - 1; i++) {
+        shouldSwitch = false;
+        if (b[i].innerHTML.toLowerCase() > b[i + 1].innerHTML.toLowerCase()) {
+          shouldSwitch = true;
+          break;
+        }
+      }
+      if (shouldSwitch) {
+        b[i].parentNode.insertBefore(b[i + 1], b[i]);
+        switching = true;
+      }
+    }
+  }
+
```

Comment:
Hi Samir, 
Thank you for checking my homework, i see there are a lot of mistakes here. But i will try fix them all and avoid to do that again. This is my big problem. I am not used to checking my common mistake before i hand my homework, Its a bad practice. But i will try to avoid that in futures homework.

Best regards,

Reply:
Hi Muhannad, no problem at all, we are here to learn from each other.